src/main/java/org/opentripplanner/api/parameter/ApiRequestMode.java

```diff
@@ -1,47 +1,43 @@
 package org.opentripplanner.api.parameter;
 
-import static org.opentripplanner.model.modes.AllowedTransitMode.fromMainModeEnum;
-
 import java.util.Collection;
-import java.util.Collections;
-import java.util.Set;
-import org.opentripplanner.model.modes.AllowedTransitMode;
+import java.util.List;
 import org.opentripplanner.transit.model.network.TransitMode;
 
 public enum ApiRequestMode {
   WALK(),
   BICYCLE(),
   SCOOTER(),
   CAR(),
-  TRAM(fromMainModeEnum(TransitMode.TRAM)),
-  SUBWAY(fromMainModeEnum(TransitMode.SUBWAY)),
-  RAIL(fromMainModeEnum(TransitMode.RAIL)),
-  BUS(Set.of(fromMainModeEnum(TransitMode.BUS), fromMainModeEnum(TransitMode.COACH))),
-  FERRY(fromMainModeEnum(TransitMode.FERRY)),
-  CABLE_CAR(fromMainModeEnum(TransitMode.CABLE_CAR)),
-  GONDOLA(fromMainModeEnum(TransitMode.GONDOLA)),
-  FUNICULAR(fromMainModeEnum(TransitMode.FUNICULAR)),
-  TRANSIT(AllowedTransitMode.getAllTransitModes()),
-  AIRPLANE(fromMainModeEnum(TransitMode.AIRPLANE)),
-  TROLLEYBUS(fromMainModeEnum(TransitMode.TROLLEYBUS)),
-  MONORAIL(fromMainModeEnum(TransitMode.MONORAIL)),
+  TRAM(TransitMode.TRAM),
+  SUBWAY(TransitMode.SUBWAY),
+  RAIL(TransitMode.RAIL),
+  BUS(TransitMode.BUS, TransitMode.COACH),
```

Comment:
Just out of curiosity, why don't we have separate ApiRequestMode for coach?

Reply:
I think it is because COACH was not supported in an early version of the API, and then changing it to the "right-thing" is a breaking change. But, it is clearly wrong.